packages/expo-video/ios/Records/ProgressUpdate.swift

```diff
@@ -0,0 +1,9 @@
+import ExpoModulesCore
+
+// swiftlint:disable redundant_optional_initialization - Initialization with nil is necessary
+internal struct ProgressUpdate: Record {
+  @Field var currentTime: Double = 0
+  @Field var currentLiveTimestamp: Double? = nil
```

Comment:
Do we need to initialize with 'null'? For example, here https://github.com/expo/expo/blob/40053ee23e1c94c1a96914f2babee2db9491744e/packages/expo-camera/ios/Current/CameraRecordingOptions.swift#L4-L5, we didn't do it. I'm not sure if it will be populated with 0 or with 'null', that's why I'm asking 😅

Reply:
As far as I remember, it's a Swift limitation – if you don't explicitly provide the default value, the default struct initializer will expect a `Field<Double?>` type instead of `Double?` 😢 
For records that are used as arguments it's fine to omit the default value, but if you want to initialize the record yourself (and return it) then it seems more convenient if you don't have to do `ProgressUpdate(currentTime: Field<Double>(0))`